config/initializers/hesa_degree_types.rb

```diff
@@ -1,92 +1,86 @@
+  [57, 'B AG', 'Bachelor of Agriculture', :bachelor],
+  [58, 'B ACC', 'Bachelor of Accountancy', :bachelor],
+  [59, 'B ADMIN', 'Bachelor of Administration', :bachelor],
+  [60, 'BBA', 'Bachelor of Business Administration', :bachelor],
+  [61, 'BCS', 'Bachelor of Combined Studies', :bachelor],
+  [62, 'B COMM', 'Bachelor of Commerce', :bachelor],
+  [63, 'BD', 'Bachelor of Divinity', :bachelor],
+  [64, 'BDS', 'Bachelor of Dental Surgery', :bachelor],
+  [65, 'B ENG', 'Bachelor of Engineering', :bachelor],
+  [66, 'B ENG/BS', 'Bachelor of Engineering with Business Studies', :bachelor],
+  [67, 'BFA', 'Bachelor of Fine Art', :bachelor],
+  [68, 'BGS', 'Bachelor of General Studies', :bachelor],
+  [69, 'BH', 'Bachelor of Humanities', :bachelor],
+  [70, 'B HYG', 'Bachelor of Hygiene', :bachelor],
+  [71, 'LLB', 'Bachelor of Law', :bachelor],
+  [72, 'B LIB', 'Bachelor of Librarianship', :bachelor],
+  [73, 'BLS', 'Bachelor of Librarianship and Info. Studies', :bachelor],
+  [74, 'B LITT', 'Bachelor of Literature', :bachelor],
+  [75, 'BL ECON', 'Bachelor of Land Economy', :bachelor],
+  [76, 'B MED SC', 'Bachelor of Medical Science', :bachelor],
+  [77, 'BM', 'Bachelor of Medicine', :bachelor],
+  [78, 'B MET', 'Bachelor of Metallurgy', :bachelor],
+  [79, 'B MET/EN', 'Bachelor of Metallurgy and Engineering', :bachelor],
+  [80, 'B MU', 'Bachelor of Music', :bachelor],
+  [81, 'BN', 'Bachelor of Nursing', :bachelor],
+  [82, 'B PHARM', 'Bachelor of Pharmacy', :bachelor],
+  [83, 'B SC', 'Bachelor of Science', :bachelor],
+  [84, 'BSC/ECO', 'Bachelor of Science Economics', :bachelor],
+  [85, 'BSC/ENG', 'Bachelor of Science &amp; Engineering', :bachelor],
+  [86, 'BSC/TEC', 'Bachelor of Science &amp; Technology', :bachelor],
+  [87, 'BSC SS', 'Bachelor of Science in Social Science', :bachelor],
+  [88, 'BSC SPT', 'Bachelor of Science in Speech Therapy', :bachelor],
+  [89, 'BSS', 'Bachelor of Social Science', :bachelor],
+  [90, 'BS', 'Bachelor of Surgery', :bachelor],
+  [91, 'B CH', 'Bachelor of Chirurgiae', :bachelor],
+  [92, 'B TECH', 'Bachelor of Technology', :bachelor],
+  [93, 'B THEOL', 'Bachelor of Theology', :bachelor],
+  [94, 'B VMED', 'Bachelor of Veterinary Medicine', :bachelor],
+  [95, 'BVMS', 'Bachelor of Veterinary Medicine and Surgery', :bachelor],
+  [96, 'B VSC', 'Bachelor of Veterinary Science', :bachelor],
+  [97, 'B ED', 'Bachelor of Education Scotland &amp; N Ireland', :bachelor],
+  [98, 'B PHIL', 'Bachelor of Philosophy', :bachelor],
+
+  [200, 'MA', 'Master of Arts', :master],
+  [201, 'M LIB', 'Master of Librarianship', :master],
+  [202, 'M LITT', 'Master of Literature', :master],
+  [203, 'M MUS', 'Master of Music', :master],
+  [204, 'M PHIL', 'Master of Philosophy', :master],
+  [205, 'M SC', 'Master of Science', :master],
+  [206, 'M THEOL', 'Master of Theology', :master],
+  [207, 'Certificate of Membership of Cranfield Institute of Technology', :master],
```

Comment:
Missing second field?

```suggestion
  [207, nil, 'Certificate of Membership of Cranfield Institute of Technology', :master],
```

Reply:
Good spot, thanks :+1: